.project-inventory.json

```diff
@@ -0,0 +1,6 @@
+{
+  "classification": "cli-application",
+  "jiraKey": "ICOMP",
+  "jiraComponent": "",
```

Comment:
I we use `IDEVT` as JIRA key, then this should be `php`.

Reply:
Updated the component to php.